source/develop/release-management/features/network/allowExplicitDnsConfiguration.html.md

```diff
@@ -0,0 +1,176 @@
+    <xs:sequence>
+      <xs:element maxOccurs="1" minOccurs="0" name="name_servers">
+        <xs:complexType>
+          <xs:annotation>
+            <xs:appinfo>
+              <jaxb:class name="NameServersList"/>
+            </xs:appinfo>
+          </xs:annotation>
+          <xs:sequence>
+            <xs:element maxOccurs="unbounded" minOccurs="0" name="name_server" type="xs:string">
+              <xs:annotation>
+                <xs:appinfo>
+                  <jaxb:property name="NameServers"/>
+                </xs:appinfo>
+              </xs:annotation>
+            </xs:element>
+          </xs:sequence>
+        </xs:complexType>
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+```
+
+After doing so, we're prepared to use following xml fragment defining
+DNS configuration in several places (read on):
+
+```
+<dns_resolver_configuration>
+  <name_servers>
+    <name_server>1.1.1.1</name_server>
+    <name_server>2.2.2.2</name_server>
+  </name_servers>
+</dns_resolver_configuration>
+```
+
+#### Updating Management Network
+only management network can be updated with DNS configuration.
+
+```
+PUT /ovirt-engine/api/networks/{network:id} HTTP/1.1
+Content-Type: application/xml
+
+<network>
+  <dns_resolver_configuration>
+    <name_servers>
+      <name_server>1.1.1.1</name_server>
+      <name_server>2.2.2.2</name_server>
+    </name_servers>
+  </dns_resolver_configuration>
+</network>
+```
+
+#### Updating 'network attachment' of Management Network on specific Host 
+You can update any network with DNS Configuration, however all such DNS
+Configurations will be simply ignored during creation of new host except 
+for the DNS configuration defined on the network, which happens to be 
+management network at time of creating new host.
+
+```
+PUT ovirt-engine/api/hosts/{host:id}/networkattachments/{networkattachment:id} HTTP/1.1
+Content-type: application/xml
+
+<network_attachment>
+  <dns_resolver_configuration>
+    <name_servers>
+      <name_server>1.1.1.1</name_server>
+      <name_server>2.2.2.2</name_server>
+    </name_servers>
+  </dns_resolver_configuration>
+</network_attachment>
+
+```
+
+### GUI
+
+As mentioned, you can specify DNS Configuration at two places:
+
+#### Updating (Management) Network
+You can update any network with DNS Configuration, however all such DNS
+Configurations will be simply ignored during creation of new host except 
+for the DNS configuration defined on the network, which happens to be 
+management network at time of creating new host.
+![Edit Logical Network Dialog with DNS Configuration](editLogicalNetworkDialogWithDnsConfiguration.png "Edit Logical Network Dialog with DNS Configuration")
+
+#### Updating 'attachment' of Management Network on specific Host 
+![Editing Network Attachment Dialog with DNS configuration](editNetworkAttachmentDialogWithDnsConfiguration.png "Editing Network Attachment Dialog with DNS configuration")
```

Comment:
Can we please add 'out-of-sync' scenario for manual changing of the resolv.conf under the engine's legs.
For example: Changing the name server in resolv.conf and then doing refresh caps in UI should end up in 'out-of-sync' management network. This should be applied also on multi-host command.